environments/development/common/rds.tf

```
@@ -15,11 +15,16 @@ module "postgres" {
 
   allocated_storage     = 20
   max_allocated_storage = 40
+  security_group_ids    = [data.aws_security_group.be_to_rds_security_group.id]
```

```suggestion
  security_group_ids    = [module.alb-security-group.be_to_rds_security_group_id]
```

We can just use that directly here

---

I looked through your commits and it looks like you just passed it as a string, it should be in a list. I have this working locally, should I commit this change?